app/views/mods_panel.py

```diff
@@ -72,7 +72,8 @@ def uuid_to_mod_name(uuid: str) -> str:
     Returns:
         str: The name of the mod corresponding to the UUID.
     """
-    return MetadataManager.instance().internal_local_metadata[uuid]["name"].lower()
+    name = MetadataManager.instance().internal_local_metadata[uuid]["name"]
+    return name.lower() if name is not None else ""
```

Comment:
Would it be better to set the name as something like `unnamed mod` rather than a blank string?

Reply:
If name is a blank string, it will be at the top of the mod list by default (sort by name), if name is "unnamed mod", it will be in the middle of the mod list. But I can change that if you want.

Reply:
Good point. Just worried if a blank element in the list will cause confusion for some. Thoughts on something like #UnknownMod?